torch/csrc/jit/runtime/graph_executor.cpp

```diff
@@ -131,7 +131,7 @@ struct CaptureList {
       auto tensors = val.toTensorList();
       sizes_.push_back(tensors.size());
 
-      for (const at::Tensor& tensor : tensors) {
```

Comment:
Why removing ref?

Reply:
As per clang-tidy, this is not a ref - there is an implicit conversion to `Tensor` here that this makes more explicit. I think it's related to the fact that `tensors` is a `c10::List` and a range based for actually gives you `ListElementReference` and not `Tensor`.